collect_app/src/test/java/org/odk/collect/android/preferences/ChangeAdminPasswordDialogTest.java

```diff
@@ -0,0 +1,141 @@
+package org.odk.collect.android.preferences;
+
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.text.InputType;
+import android.widget.CheckBox;
+import android.widget.EditText;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.FragmentActivity;
+import androidx.fragment.app.FragmentManager;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.R;
+import org.odk.collect.android.support.RobolectricHelpers;
+import org.odk.collect.android.support.TestActivityScenario;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.shadows.ShadowDialog;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertFalse;
+import static org.odk.collect.android.preferences.AdminKeys.KEY_ADMIN_PW;
+import static org.odk.collect.android.preferences.AdminPreferencesActivity.ADMIN_PREFERENCES;
+import static org.robolectric.Shadows.shadowOf;
+
+@RunWith(RobolectricTestRunner.class)
+public class ChangeAdminPasswordDialogTest {
+
+    private TestActivityScenario<DialogFragmentTestActivity> activityScenario;
```

Comment:
I don't think the `activityScenario` needs to be a field here as you're only using `ActivityScenario` in one test. I think just make it a local variable in there.